todo-mvc-tsx/README.md

```diff
@@ -0,0 +1,76 @@
+# Dojo2 TodoMVC Example
+
+A reference implementation of [TodoMVC](http://todomvc.com/) built with current pre-release versions of Dojo 2 packages.
+
+- [Pre-requisites](#pre-requisites)
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
+
+`@dojo/cli` must be installed globally:
+
+```shell
+npm install @dojo/cli -g
+```
+
+## Usage
+
+To use `todo-mvc`, clone the `examples` repository and install the npm dependencies:
+
+```shell
+git clone https://github.com/dojo/examples
+cd examples/todo-mvc
+npm install
+```
+
+To run the application simply run the dojo build.
+
+```
+dojo build
+```
+
+To watch and serve the application source, then run the build with the `-w` command.
+
+```
+dojo build -w
+```
+
+## Features
+
+This package demonstrates how to build a simple application using Dojo 2 widgets.
+
+It is an example of:
+
+* Custom widgets using `@dojo/widget-core`
+* Applying CSS to widgets 
+* Using `@dojo/router`
+
+## How do I contribute?
+
+We appreciate your interest!  Please see the [Dojo Meta Repository](https://github.com/dojo/meta#readme) for the
+Contributing Guidelines and Style Guide.
+
+### Installation
+
+To start working with this package, clone the repository and run `npm install`.
+
+In order to build the project run `dojo build`.
+
+### Testing
+
+Test cases MUST be written using [Intern](https://theintern.github.io) using the Object test interface and Assert assertion interface.
```

Comment:
Note we say this in our guidelines, but the tests for todomvc (not just this version) use the bdd and describe rather than Object test interface, so perhaps we should relax these guidelines?